### PR TITLE
doc/*.txt -> *.rst

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by mr-c** [comment]
Will close #910

**Comment by mr-c** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by ctb** [comment]
Can this branch be tested on RTD first?

**Comment by mr-c** [comment]
@ctb Sure, I'll do a special build there

**Comment by ctb** [comment]
On Sun, Apr 05, 2015 at 04:14:51AM -0700, Michael R. Crusoe wrote:
> @ctb Sure, I'll do a special build there
thanks - and good catch!

**Comment by mr-c** [comment]
@ctb See http://khmer.readthedocs.org/en/doc-extension-rename/

**Comment by mr-c** [comment]
Ready for review and merge

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,12 @@
 2015-04-05  Michael R. Crusoe  <mcrusoe@msu.edu>
 
+   * doc/*/*.txt: all documentation sources have been renamed to use the rst
+   extension to indicate that they are ReStructuredText files. This enables
```
Comment (ctb) [inline ChangeLog]: Note, it's reStructuredText, no capital 'R' at the beginning.

File: doc/requirements.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 http://athyra.ged.msu.edu/~mcrusoe/autoprogram/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
+setuptools==3.4.1
```
Comment (ctb) [inline doc/requirements.txt]: Please add this info to the changelog.

Comment (ctb) [inline doc/requirements.txt]: i..e the fact that requirements.txt was changed.

File: doc/requirements.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 http://athyra.ged.msu.edu/~mcrusoe/autoprogram/sphinxcontrib-autoprogram-0.1.1-khmerdev-20140331.tar.gz#egg=sphinxcontrib-autoprogram==0.1.1-khmerdev-20140331
```
Comment (ctb) [inline doc/requirements.txt]: This URL should probably be moved off of athyra (and MSU)...

File: doc/conf.py [hunk 4]
```diff
@@ -56,16 +57,16 @@
...
-try:
-    release = pkg_resources.get_distribution('khmer').version
-except pkg_resources.DistributionNotFound:
-    print 'To build the documentation, The distribution information of khmer' \
-        ' has to be available.  Either install the package into your' \
-        ' development environment or run "setup.py develop" to setup the' \
-        ' metadata.  A virtualenv is recommended!'
-    sys.exit(1)
-del pkg_resources
+sys.path.insert(0, '../')
```
Comment (ctb) [inline doc/conf.py]: Please comment on this in the ChangeLog.

Comment (ctb) [inline doc/conf.py]: Ahh, sorry, I see that you have already, NM :)

**Comment by ctb** [comment]
On Sun, Apr 05, 2015 at 04:31:56AM -0700, Michael R. Crusoe wrote:
> @ctb See http://khmer.readthedocs.org/en/doc-extension-rename/

BTW, this looks good to me - go ahead and merge when ready.

File: doc/requirements.txt (hunk 3 above)
Comment (mr-c) [inline doc/requirements.txt]: Noted: https://github.com/ged-lab/khmer/issues/913

File: doc/requirements.txt (hunk 2 above)
Comment (mr-c) [inline doc/requirements.txt]: Done

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**